specifyweb/frontend/js_src/lib/components/WorkBench/CellMeta.ts

```diff
@@ -61,7 +61,7 @@ export class WbCellMeta {
 
   public constructor(private readonly wbView: WbView) {
     this.updateCellInfoStats = throttle(
-      () => this.updateCellInfoStats(),
+      () => this.updateCellInfoStats.bind(this),
```

Comment:
thanks for catching this. one cool thing I recently learned about TypeScript, is that if you replace your property definition from this style:

```ts
  private name(): type {}
```

to this style:

```ts
  private name = (): type => {};
```

then TypeScript takes adds this to the constructor() automatically for you:

```ts
this.name = this.name.bind(this);
```

which means you don't have to add .bind() manually anywhere anymore

(but of course an even better solution is to not use classes  😋 as that's the only way to avoid all this-related bugs and all the other unholy things that come with OOP style)

cc @melton-jason